src/axolotl/utils/schemas/config.py

```diff
@@ -109,6 +109,12 @@ class AxolotlInputConfig(
             "description": "Don't upcast the embeddings to float32 when using PEFT. Useful for low-VRAM GPUs"
         },
     )
+    reinit_weights: bool | None = Field(
+        default=None,
+        json_schema_extra={
+            "description": "Reinitialize model weights randomly instead of loading pretrained weights"
+        },
+    )
```

Comment:
Didn't realize we never had `random_init_weights` in our schema.

Reply:
Wonder if we need to add an alias to it for those who may have used it.

Reply:
It got deleted by some merge I think. We have to be careful to have PRs properly rebased before merge